codalab/lib/bundle_cli.py

```diff
@@ -1276,6 +1276,12 @@ def do_upload_command(self, args):
                 '--output-path',
                 help='Path to download bundle to.  By default, the bundle or subpath name in the current directory is used.',
             ),
+            Commands.Argument(
+                '-f',
+                '--force',
```

Comment:
Would be semantically more accurate to call this `--overwrite`, but alas, `-o` is taken...